after/syntax/html.vim

```diff
@@ -0,0 +1,48 @@
+if srcery#helper#ShouldAbort('html')
+  finish
+endif
+
+let s:none = g:srcery#palette.none
+let s:black = g:srcery#palette.black
+let s:bright_white = g:srcery#palette.bright_white
+
+let s:bold = g:srcery#palette.bold
+let s:italic = g:srcery#palette.italic
+let s:underline = g:srcery#palette.underline
+
+" HTML: {{{
+
+hi! link htmlTag SrceryBlue
```

Comment:
Same as line 19?

@roosta Thoughts on what's correct?

Reply:
Hum... Should probably be a separate issue. This is a refactor, I shouldn't change behaviour in general 😄

Reply:
Its supposed to be `hi! link htmlTag SrceryBrightBlack`, and its always been the last definition that took effect, I see no problem with removing line 15 since it never actually worked in the first place.

Reply:
I'm happy to merge today if we fix this. 😄

Reply:
What about the end tag? It's set to blue as well.

Reply:
Its also supposed to be SrceryBrightBlack, weird I've never noticed that :D

Reply:
If you prefer I can make a separate commit for that, seeing as it is a bug fix

Reply:
I would rather have that since it might be hard to trace back stuff... This is a big commit already 😄

Reply:
I'll do a separate html fix later, marking this as resolved